bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/ApiBridgeHandler.java

```diff
@@ -162,6 +162,28 @@ public void initialize() {
     }
 
     public void openConnection(@Nullable String code, @Nullable String redirectUri) {
+        authenticate(code, redirectUri);
+
+        logger.debug("Connecting to Netatmo API.");
+
+        ApiHandlerConfiguration configuration = getConfiguration();
+        if (!configuration.webHookUrl.isBlank()) {
+            SecurityApi securityApi = getRestManager(SecurityApi.class);
+            if (securityApi != null) {
+                WebhookServlet servlet = new WebhookServlet(this, httpService, deserializer, securityApi,
```

Comment:
You should first dispise the servlet before creating a new one?

Reply:
Same situation as above, this is a lack of further enhancements, not something introduced.

Reply:
It looks easy to fix. Could be done in this PR.